storage/remote/otlptranslator/prometheusremotewrite/metrics_to_prw.go

```diff
@@ -121,6 +126,7 @@ func (c *PrometheusConverter) FromMetrics(ctx context.Context, md pmetric.Metric
 		WithMetricSuffixes: settings.AddMetricSuffixes,
 		UTF8Allowed:        settings.AllowUTF8,
 	}
+	unitNamer := otlptranslator.UnitNamer{}
```

Comment:
```suggestion
	unitNamer := otlptranslator.UnitNamer{UTF8Allowed: settings.AllowUTF8}
```

Reply:
The translator package already has unit tests for it, but adding a few here wouldn't hurt I guess 🤷